packages/solarwinds-apm/src/config.ts

```diff
@@ -166,6 +166,7 @@ const schema = z.object({
       swMetrics: boolean.default(!IS_SERVERLESS),
       initMessage: boolean.default(!IS_SERVERLESS),
       resourceDetection: boolean.default(true),
+      instrumentationsDefaultDisabled: boolean.default(IS_SERVERLESS),
```

Comment:
Dumb question: is the the main toggle between lambda and regular loading?

Reply:
`IS_SERVERLESS` is yeah, `instrumentationsDefaultDisabled` just toggles the default behaviour between instrumentations being disabled or enabled by default